app/data/livechat/cache-strategy/ILiveChatRemoteDataSource.ts

```diff
@@ -7,6 +7,6 @@ export default interface ILiveChatRemoteDataSource {
 
     createVisitor(visitor: IVisitor): Promise<IVisitor>;
 
-    createRoom(visitor: IVisitor): Promise<ILivechatRoom>;
+    createRoom(visitor: IVisitor, department: Department): Promise<ILivechatRoom>;
```

Comment:
Is this department param necessary? https://rocketchat.github.io/Rocket.Chat.Apps-engine/interfaces/ivisitor.html#department